### DISCUSSION TITLE
Replicate function for some of the entities in the system

### DISCUSSION THREAD

**Discussion by mehmetkuzu** [bounswe/bounswe2023group2#62] [Ideas]
Something like this:
Under resource (1.2.3.1.)
Users shall be able to replicate a created resource 
Needs
Users shall be able to replicate a created need

Profile (May be in the admin section) 
Admins shall be able to copy profile information of a user to another user

**Comment by mehmetkuzu** [comment]
Check this.
No comment is yes...

**Comment by mehmetkuzu** [comment]
Done.
Added to the document.
Someone review and close the issue.

**Comment by cahidenes** [comment]
I am sorry that I am seeing this discussion late.
Are these requirements necessary? I think we should avoid adding too many requirements to make our lives easier.

**Comment by mehmetkuzu** [comment]
I see.
We can (for now) use "should" instead of shall (must).

We should also try to make the users' lives easier. This kind of "template" and "copy/paste" operations can be very helpful. (For recurring entries by the user etc.)

**Reply by cahidenes** [reply]
Does "should" mean "we have the option to implement this requirement or not"? If it is, "should" is OK.

**Reply by mehmetkuzu** [reply]
Exactly. (As far as I understood)
